src/bundles/SellerRegistration/components/Signup/Signup.js

```diff
@@ -112,6 +112,9 @@ class Signup extends React.Component {
       actions.stepNextPersist(this.nextStep.pattern, this.step);
     },
     onSubmitFailed: (e) => {
+      if (e && 'preventDefault' in e) {
```

Comment:
would there be a case where e is not an event?

Reply:
I wonder if it ever is an event... https://github.com/davidkpiano/react-redux-form/blob/8e8a375f02c1a978bf1804b2cc5a5413e96bf1b4/src/components/form-component.js#L296

Reply:
If it's not, do we need this?